Terminal.Gui/Terminal.Gui.csproj

```diff
@@ -17,27 +17,32 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/migueldeicaza/gui.cs/</PackageProjectUrl>
     <PackageTags>csharp, terminal, c#, f#, gui, toolkit, console</PackageTags>
-    <Description>Console-based user interface toolkit for .NET applications.</Description>
+    <Description>Console user interface toolkit for .NET applications.</Description>
     <Owners>Miguel de Icaza</Owners>
     <Summary>Application framework for creating modern console applications using .NET</Summary>
     <Title>Terminal.Gui is a framework for creating console user interfaces</Title>
     <PackageReleaseNotes>
-      0.90: Feature Complete pre-release of 1.00
-      * API documentation completely revamped and updated. Readme upated. Contributors guide added (Thanks @tig!)
+      This (0.90) is the "Feature Complete" pre-release of Terminal.Gui (aka gui.cs) 1.0. This release is a signficant upgrade from the previous published release (0.81). Most of the major changes and bug fixes are listed below. NOTE: This release includes breaking changes to the API; we will strive to avoid any more breaking changes before 1.0.
+
+      What's new:
       * New sample/demo app - UI Catalog - Replaces demo.cs with an easy to use and extend set of demo scenarios. (Thanks @tig!)
+      * The API documentation is completely revamped and updated. Readme upated. Contributors guide added (Thanks @tig!)
       * MenuBar can now have MenuItems directly (enables top-level menu items with no submenu). (Thanks @tig!)
-      * Apps can now get KeyUp/KeyDown events.  (Thanks @tig!)
+      * API semantics are much more consistent across classes. For example, all events are now defined in terms of Action instead of EventHanlder. BREAKING CHANGE. (Thanks @bdisp and @tig!)
+      * The project has been refactored an reorganized to reduce risk of bugs and make it easier to c
+      * Symbols are now included with the nuget package.
+
+      Fixes/Improvements (partial list; see Github issues for complete list):
```

Comment:
Probably worth mentioning that we are now using `event`s instead of just actions btw? Not sure if this is appropriate, but this is definitely a breaking change that might be worth documenting.